### PR TITLE
[Fixes #5260] Updates DSC Resources Article

### LINKED ISSUE #5260: Clarify "community resources", the state of the DscResources GitHub repository and more
The section "[Built-in resources](https://docs.microsoft.com/en-us/powershell/scripting/dsc/resources/resources?view=powershell-6#built-in-resources)" states:

> In addition to community resources, there are built-in resources for Windows, resources for Linux, and resources for cross-node dependency.

As a technical person trying to understand the DSC big picture I have the following questions after reading the above statement:

1. What are those "_community resources_"? They are mentioned here casually without linking somewhere. Is there a well-known set of community resources? Or is this rather a generic term referring to any resource that might have been created by a third party?
2. "_there are **built-in** resources for Windows, resources for Linux, and resources for cross-node dependency_" - what is "built-in" and what is "not built-in"? Does the term "built-in" also refer to Linux? In the sub-heading of the listed resources it says "Linux resources", "built-in" is missing. In the resource reference left navigation node it says "Linux built-in resources" instead. This should be made consistent. Why am I harping on this? Because "built-in" for me means: I don't have to do anything. "Not built-in" means: I have to install something. (Is this even right? What exactly is built into what?)
2. How does the GitHub repository [DscResources](https://github.com/PowerShell/DscResources) fit into the picture? The repo readme states that this is a list of resources from Microsoft. Is it equivalent to the "built-in" resources? Is this the "community resources"? Is it a different set of resources? And if yes, do I have to differentiate between the ["non-x"-resources](https://github.com/PowerShell/DscResources/tree/master/DscResources) and the ["x"-resources](https://github.com/PowerShell/DscResources/tree/master/xDscResources) (those were kind of beta if I remember correctly)?
3. Looking also at [Azure Automation State Configuration (DSC)](https://docs.microsoft.com/en-us/azure/automation/automation-dsc-overview) I wonder which resources are available there in Azure. Is it the "built-in" ones? The GitHub repo ones? ...? Since the topics are somewhat connected I wished there was a hint in the right direction here.
4. And a bit fringy but also connected: what's the deal with the [Office365DSC](https://github.com/microsoft/Office365DSC) resource? Is there a connection to the aforementioned topics or is this a project completely separated from it all?

My goal is to
- learn about the available resources for DSC
- know which DSC resources are from Microsoft and which ones are not
- learn about non-MS resources from third parties or the community (if there are known places)
- know which resources I can expect to work in Azure with Azure Automation State Configuration (DSC)

Clarifying the above questions would help a lot.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e8a198a3-6fcd-624e-2fc4-deb7e1dcb3e6
* Version Independent ID: d3515939-e925-2b6b-b31f-c212440f7d47
* Content: [DSC Resources - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/dsc/resources/resources?view=powershell-6)
* Content Source: [reference/docs-conceptual/dsc/resources/resources.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/dsc/resources/resources.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary

Update information to better communicate resources type and availability.

## PR Context

Fixes #5260 
Fixes [AB#1682540](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1682540)

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [x] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/dsc/resources/resources.md [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
-ms.date:  12/12/2018
+ms.date:  02/28/2020
 keywords:  dsc,powershell,configuration,setup
 title:  DSC Resources
 ---
 
 # DSC Resources
 
->Applies To: Windows PowerShell 4.0, Windows PowerShell 5.0
+> Applies To: Windows PowerShell 4.0, Windows PowerShell 5.x, and PowerShell
```
Comment (sdwheeler) [inline reference/docs-conceptual/dsc/resources/resources.md]: "and PowerShell" what?

File: reference/docs-conceptual/dsc/resources/resources.md [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
+with the `-Syntax` parameter to get cmdlet syntax. The output you see will show the template used
+for a resource block for the resource you specify.
 
 ```powershell
 Get-DscResource -Syntax Service
 ```
 
-The output you see should be similar to the output below, though this resource's syntax could change in the future. Like cmdlet syntax, the *keys* seen in square brackets, are optional. The types specify the type of data each key expects.
+The output you see should be similar to the output below, though this resource's syntax could change
+in the future. Like cmdlet syntax, the *keys* seen in square brackets, are optional. The types
```
Comment (sdwheeler) [inline reference/docs-conceptual/dsc/resources/resources.md]: use underscores for italics

Comment (chasewilson) [reply reference/docs-conceptual/dsc/resources/resources.md]: Fixed

File: reference/docs-conceptual/dsc/resources/resources.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/dsc/resources/resources.md]: Updated

**Pull request merged**